packages/backend-app-api/src/wiring/BackendInitializer.ts

```diff
@@ -407,6 +407,26 @@ export class BackendInitializer {
       // The startup failed, but we may still want to do cleanup so we continue silently
     }
 
+    // Get all plugins.
+    const pluginMap = new Map<string, boolean>();
```

Comment:
```suggestion
    const pluginMap = new Map<string, true>();
```

or. making it a `Set<string>` and doing `[...s].map(` below